topics/data-structure/index.md

```diff
@@ -0,0 +1,8 @@
+---
+aliases: data-structures
+display_name: Data structure
+short_description: Data structure is way of organizing and storing data.
+topic: data-structure
```

Comment:
Could you rename this and the directory to `data-structures`? It's the more popular topic, see https://github.com/topics/data-structures with 2,650 repositories versus https://github.com/topics/data-structure with 302.

Then you can check off data-structures in the topics-todo.md file in this branch.